.gitignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 npm-debug.log
 package-lock.json
+yarn.lock
```

Comment:
`package-lock.json` and `yarn.lock` should be committed in the project, I added `yarn.lock` since you had already ignored the `package-lock.json` but I do think both of these files should be in the repo